spec/rubocop/cop/style/redundant_sort_spec.rb

```diff
@@ -231,12 +231,12 @@
     expect_no_offenses('[[1, 2], [3, 4]].first.sort')
   end
 
-  # `[3, 1, 2].sort_by(&:size).last` is equivalent to `[3, 1, 2].max_by(&:size)`.
+  # `[2, 1, 3].sort_by(&:size).last` is not equivalent to `[2, 1, 3].max_by(&:size)`.
   it 'does not register an offense when using `sort_by(&:size).last`' do
     expect_no_offenses('[2, 1, 3].sort_by(&:size).last')
   end
 
-  # `[3, 1, 2].sort_by { |i| i.size }.last` is equivalent to `[3, 1, 2].max_by { |i| i.size }`.
+  # `[2, 1, 3].sort_by { |i| i.size }.last` is not equivalent to `[2, 1, 3].max_by { |i| i.size }`.
```

Comment:
I noticed that these comments had a typo so I figured I'd fix them while I'm looking at it.